ballerina/listener.bal

```diff
@@ -60,6 +61,17 @@ public class Listener {
         if result is error {
             return error Error("Error occurred while attaching the service", result);
         }
+
+        if graphiql is GraphiQL && graphiql.enable {
+            check validateGraphiQLPath(graphiql.path);
+            string gqlServiceBasePath = name is () ? "" : getBasePath(name);
+            HttpService graphiqlService = getGraphiQLService(serviceConfig, gqlServiceBasePath);
+            attachGraphiQLServiceToGraphqlService(s, graphiqlService);
+            result = self.httpListener.attach(graphiqlService, graphiql.path);
+            if result is error {
+                return error Error("Error occured while attaching the GraphiQL service", result);
```

Comment:
"Error occured while attaching the GraphiQL service" -> "Error occured while attaching the GraphiQL endpoint" ?

Reply:
occured -> occurred